pkg/domain/infra/abi/containers.go

```diff
@@ -77,10 +78,31 @@ func getContainersAndInputByContext(all, latest bool, names []string, filters ma
 			}
 		}
 	case latest:
-		ctr, err = runtime.GetLatestContainer()
-		if err == nil {
-			rawInput = append(rawInput, ctr.ID())
-			ctrs = append(ctrs, ctr)
+		isPod := false
+		for _, name := range names {
```

Comment:
It feels like it would be safer to add a `pod` parameter to the function signature, instead of trying to interpret input to determine if the user wanted a pod?

Reply:
You're right. I can change that, and since `PodLogs` doesn't call `getContainersByContext` directly, I think I will need to add some logic in `ContainerLogs1` to iterate over the names, do a lookup of the container, and see if one of them is an infra container. Unless there is an easier way to do it.